.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -32,11 +32,11 @@ body:
       description: select all version where you have experienced this issue
       multiple: true
       options:
-        - "v1_6"
-        - "v1_7"
-        - "v1_8"
-        - "v1_9"
-        - "v2_0"
+        - "v1.6"
```

Comment:
Don't we need to update the version set in https://github.com/Lightning-AI/lightning/blob/master/.github/workflows/labeler-issue.yml#L31?

Reply:
Since you are pinned to the major version (`v2`), you don't have to update it unless a new major version is released.
The latest version is `v2.0.6` - https://github.com/redhat-plumbers-in-action/advanced-issue-labeler/releases